application/views/options/oqrs.php

```diff
@@ -66,4 +67,5 @@
 		</div>
 	</div>
 
-</div>
\ No newline at end of file
+</div>
+<?php } >
```

Comment:
incomplete PHP closing

```suggestion
<?php } ?>
```